### ISSUE TITLE
Config rewriting

### ISSUE THREAD

**Issue by Disketaa** [MehVahdJukaar/amendments#145]
The value is changing every time, and not consistent 
![image](https://github.com/user-attachments/assets/eeb1fabb-a710-4bcc-b69f-fe73379421e8)

**Comment by MehVahdJukaar** [comment]
i need latest.log

**Comment by Disketaa** [comment]
[latest.log](https://github.com/user-attachments/files/17369831/latest.log)

**Comment by MehVahdJukaar** [comment]
didnt seem like it corrected itself there. is whole config resetting?

**Comment by Disketaa** [comment]
No, just a few of the values. As I noticed, they changed after the mod update, but the changes are weird. Like from 0.00000123 to 0.00000132 (for example)

**Comment by MehVahdJukaar** [comment]
do they reset to their value? or is it just rounding up the value you give them?
